src/Modules/Dashboard/Dashboard.module.css

```diff
@@ -0,0 +1,19 @@
+
+.dashboardContainer {
+    .bgCard {
+        background-color: rgba(248, 249, 251, 1);
```

Comment:
We need to avoid using fixed colors in this projects as possible and use css variables instead. In this project, we will need to implement dark mode functionality, in which, we will need to have our light and dark colors set. Having colors fixed, we will make this a nightmare and we will need to change every element manually.

With CSS variable, it will be as simple as this:

```
.dark{
--bg-main: black;
other css variables
}

.light{
--main:white;
other css variables
}
```

Reply:
sure , will use css variables